### PR TITLE
build: update packages

### PR DESCRIPTION
**Pull request by kellyjosephprice**
[][demo] |
:---:|

## 🧰 Changes

Updates a bunch of packages minor versions.

The snapshot updates are from puppeteer, it looks text or line height is being calculated slightly differently. I don't think this will effect the outputted code.

## 🧬 QA & Testing

- [ ] tests pass?

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@readme/emojis": "^4.0.0",
-    "@readme/syntax-highlighter": "^10.13.0",
+    "@readme/syntax-highlighter": "^10.13.2",
```
Comment (erunion) [inline package.json:32]: You might have that codemirror problem again with this one when you pull it over into the site, I'm not sure. Hopefully not.

File: package.json [hunk 2]
```diff
@@ -60,51 +60,51 @@
...
-    "babel-jest": "^27.2.1",
-    "babel-loader": "^8.2.2",
-    "codemirror": "^5.63.0",
-    "core-js": "^3.18.0",
-    "css-loader": "^6.3.0",
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
+    "babel-jest": "^27.4.0",
+    "babel-loader": "^8.2.3",
+    "codemirror": "^5.64.0",
```
Comment (erunion) [inline package.json:81]: This one might also cause problems, but hopefully not.

File: package.json [hunk 3]
```diff
@@ -60,51 +60,51 @@
...
-    "mini-css-extract-plugin": "^2.3.0",
+    "jest-puppeteer": "^6.0.2",
+    "mini-css-extract-plugin": "^2.4.5",
     "node-sass": "^6.0.1",
-    "prettier": "^2.4.1",
-    "puppeteer": "^10.1.0",
-    "rdme": "^5.1.4",
+    "prettier": "^2.5.0",
+    "puppeteer": "^12.0.1",
+    "rdme": "^5.2.0",
```
Comment (erunion) [inline package.json:96]: There's a v6 out now if you want to update this again.

**Review by erunion — APPROVED** [review]
sorry for the untethered to a review comments. looks ok though

File: package.json (hunk 1 above)
Comment (kellyjosephprice) [reply package.json:32]: Oh yea, totally forgot about that.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.35.0-next.1</b>](/readmeio/markdown/releases/tag/v6.35.0-next.1)
<br>

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.35.0</b>](/readmeio/markdown/releases/tag/v6.35.0)
<br>

**Pull request merged**